packages/gcp_metrics/fields/base-fields.yml

```
@@ -0,0 +1,12 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
```

Should we import them from ECS?
```suggestion
- name: data_stream.type
  external: ecs
```

---

I just copied this file from other integration packages, but as you suggested this could be set as `external: ecs` without any issues.

These packages are missing `_dev/build/build.yml`, so I will also those files.